quiz.js

```diff
@@ -0,0 +1,104 @@
+
+// Grow your tree
+
+// Example
+
+// Here's what the pine tree should look like when you specify a height of 7, and use the asterisk character.
+
+//       *                spaces= 6, symbol= 1  (0) + 1
+//      ***               spaces= 5, symbol= 3  (1) + 2
+//     *****              spaces= 4, symbol= 5  (2) + 3
+//    *******             spaces= 3, symbol= 7  (3) + 4
+//   *********            spaces= 2, symbol= 9  (4) + 5
+//  ***********           spaces= 1, symbol= 11 (5) + 6
+// *************          spaces= 0, symbol= 13 (6) + 7
+
+
+
+// var height = document.getElementById("treeHeight").value;
+// console.log("height:", height);
+
+// var char = document.getElementById("character").value;
+// console.log("character:", char);
+
+// var tree = "";
+// console.log("tree:", tree);
+
+
+
+var heightBox = document.getElementById("treeHeight");		// target input box in HTML, make into a variable
+heightBox.addEventListener("keypress", enter);						// add an event listener that triggers
+																													// the function "enter" upon a keypress
+
+var charBox = document.getElementById("character");
+charBox.addEventListener("keypress", enter);
+
+var button = document.getElementById("submitButton");		// target the "grow" button and assign it to a variable
+button.addEventListener("click", middle);								// create an event listener that, upon the click of the button variable, runs the function "middle"
+
+
+
+function enter (e) {															// function that says "if the keyCode of the key thats pressed is 13,
+	var key = e.keyCode;														// run the function 'middle'
+	if (13 === key)	{
+		middle();
+	}
+}
+
+var treeObject = {};															// create an empty object called "treeObject"
+
+function middle () {
+	treeObject.height = heightBox.value;						// create a key in treeObject called "height" and add to it the text in the height input box
+	treeObject.character = charBox.value;						// create a key in treeObject called "character" and add to it the text in the character input box
+	buildTree(treeObject)														// run the function "buildTree" with the treeObject object
+	console.log("height:", treeObject.height);
+	console.log("character", treeObject.character);
+}
+
+
+function buildTree (x) {
+
+	var tree = "";															// create a variable "tree" that contains an empty string
+
+
+	if (x.height === "" || x.character === "") {																		// if the value of the key "height" is empty OR
+		alert("Oops! I'm missing information, Please fill out the WHOLE form.");			// if the value of the key "character" is empty
+	}																																								// when this function is called, display this alert
+
+	for (var i = 0; i < x.height; i++) {																							// use "i" to increment/decrement the key values
+		tree += " ".repeat(x.height - i - 1) + x.character.repeat((2 * i) + 1) + '\n';	// based on the index of the for loop
+	}
+
+		console.log(tree);
```

Comment:
Lines 96 and 98 should have the same indentation as the if and for statements above them